docs/api-docs/cmds.md

```diff
@@ -357,6 +389,16 @@ function reducer(state = { /* ... */ }, action) {
 }
 ```
 
+## Convenience helpers
```

Comment:
i wouldn't call these convenience helpers. They're regular commands just like setTimeout and clearTimeout. I'd rather just throw them next to setTimeout and setInterval above.